src/taipy/gui/_default_config.py

```diff
@@ -45,6 +45,7 @@
     "async_mode": "gevent",
     "change_delay": None,
     "chart_dark_template": None,
+    "client_url_prefix": "/",
```

Comment:
Why the "client_" prefix?
What would a use case where "url_prefix" has to be used, NOT on the client?

Reply:
We decided to go for "base_url"